core/arch/arm/plat-imx/registers/imx6-crm.h

```diff
@@ -2,8 +2,8 @@
 /*
  * Copyright 2017-2019 NXP
  */
-#ifndef __MX6_CRM_REGS_H__
-#define __MX6_CRM_REGS_H__
+#ifndef __MX6_CRM_H__
+#define __MX6_CRM_H__
```

Comment:
Nitpicking: should be __IMX6_CRM_H__ to better reflect the name of the file, and same below, but TBH we are not always consistent in the way we write those file guards :-/

Reply:
You're right. I will fix this.